### DISCUSSION TITLE
Global Config Location

### DISCUSSION THREAD

**Discussion by injeniero** [microsoft/prompty#155] [Ideas]
I would like to propose to define where the global config should be located, just like many command line tools work today. Most of them follow the pattern: `current_folder -> user_home_folder -> system_wide_folder`

For prompty it could be like:
- Current Folder: `.prompty`
- User Home: `$HOME/.prompty`
- System Wide: `/etc/prompty`

@sethjuarez what do you think?

**Comment by injeniero** [comment]
For the VS (and other IDEs) extensions, I would propose it to first follow the Editor specific hierarchy and then follow the default chain. For VS it would be: `workspace_config -> global_editor_config => (default chain)`

**Comment by sethjuarez** [comment]
Currently it chooses the "closest" `prompty.config` to the prompty file itself. Thoughts on that approach?
